webextensions/api/proxy.json

```diff
@@ -2,9 +2,43 @@
   "webextensions": {
     "api": {
       "proxy": {
-        "onProxyError": {
+        "onError": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/proxy/onProxyError",
```

Comment:
Will you move the MDN page to ttps://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/proxy/onError?